.chloggen/sqlserver-report-empty-when-fail.yaml

```
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: sqlserverreceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: SQL Server receiver now reports empty strings for query texts and query plans when obfuscation failed.
```

```suggestion
note: Only empty strings for query texts and query plans when obfuscation failed.
```
Nit: The changelog prefixes the `note` with the `component`, so there's no need to include the component here 👍